src/util/opensolaris.org.sections

```diff
@@ -31,7 +31,7 @@ category = Accessories,Configuration and Preferences,Games,Graphics and Imaging,
 category = Documentation,File Managers,Libraries,Localizations,Scripts,Sessions,Theming,Trusted Extensions,Window Managers
 
 [Development]
-category = C,C++,Databases,Distribution Tools,Editors,Fortran,GNOME and GTK+,GNU,High Performance Computing,Integrated Development Environments,Java,Objective C,Observability,Other Languages,PHP,Perl,Python,Ruby,Smalltalk,Source Code Management,Suites,System,X11
+category = Assembly language,C,C++,Databases,Distribution Tools,Editors,Erlang,Fortran,GNOME and GTK+,GNU,Go,Haskell,High Performance Computing,Integrated Development Environments,Java,JavaScript,Julia,Lisp,Lua,Objective C,Observability,OCaml,Other Languages,PHP,Perl,Python,Ruby,Rust,Smalltalk,Source Code Management,Suites,System,Tcl,X11
```

Comment:
Assembler?
Where do you want to use it?

Reply:
We have nasm, yasm, etc... We always default to "Other Languages".

Reply:
I'm fine with this change, but I honestly think it's no-op...

Reply:
Sure I know it is just a bit of cosmetics :)

Reply:
It is like having "Desktop (GNOME)" as category... it annoys me for some reason :S